banned-club.sql

```diff
@@ -1603,6 +1619,7 @@ VALUES
     ('thirteen years old'),
     ('thirteen yo'),
     ('this faggot can throw hands'),
+    ('threat to national security'),
```

Comment:
HOW this ended up here

Reply:
Lots of banned clubs come from reports of users' profiles. This is one of those clubs. While I realize most users joined this club ironically, you'd be surprised what gets reported sometimes.